### DISCUSSION TITLE
Confidence intervals?

### DISCUSSION THREAD

**Discussion by cdeterman** [online-ml/river#713] [Q&A] [answered]
I am interesting in possibly using river for a time series forecasting problem but part of the requirements for the project include the need for confidence/prediction intervals.  Does river facilitate this for a time series forecasting problem?

**Comment by raphaelsty** [comment, marked as answer]
Hi @cdeterman,

You might be interested in one example in particular from the River documentation that use prediction intervals, [Quantile Regression Uncertainty](https://riverml.xyz/latest/examples/quantile-regression-uncertainty/) and especially quantile loss `optim.losses.Quantile(alpha=0.95)`. You can use this loss with most of the River models. 🙂
